### PR TITLE
Controllers: Adding the velocity information to the pose coming from the Hydra

### PR DESCRIPTION
**Pull request by samcake**
- Adding the velocity and angular Velocity information in the poses for the left and right hand Pose for the hydra.
- Removed the hydraMove.js since it s completely superseeded by the new controller graph and grabbing features

- fixed the clap.js to work and check that the velocity was passed correctly

### CODE REVIEW

File: examples/clap.js [hunk 1]
```diff
@@ -71,8 +71,12 @@ function maybePlaySound(deltaTime) {
     var palm2Position = MyAvatar.getRightPalmPosition();
     var distanceBetween = Vec3.length(Vec3.subtract(palm1Position, palm2Position));
 
-    var palm1Velocity = Controller.getSpatialControlVelocity(1);
-    var palm2Velocity = Controller.getSpatialControlVelocity(3);
+    var palmPose = Controller.getPoseValue(Controller.Standard.LeftHand);
```
Comment (jherico) [inline examples/clap.js]: unused variable?

File: libraries/input-plugins/src/input-plugins/SixenseManager.cpp [hunk 2]
```diff
@@ -443,8 +445,20 @@ void SixenseManager::handlePoseEvent(glm::vec3 position, glm::quat rotation, boo
     // TODO: find a shortcut with fewer rotations.
     rotation = _avatarRotation * postOffset * glm::inverse(sixenseToHand) * rotation * preOffset * sixenseToHand;
 
+    glm::vec3 velocity(0.0f);
+    glm::quat angularVelocity;
 
-    _poseStateMap[left ? controller::StandardPoseChannel::LEFT_HAND : controller::StandardPoseChannel::RIGHT_HAND] = controller::Pose(position, rotation);
+    if (prevPose.isValid() && deltaTime > std::numeric_limits<float>::epsilon()) {
+        velocity = (position - prevPose.getTranslation()) / deltaTime;
```
Comment (jherico) [inline libraries/input-plugins/src/input-plugins/SixenseManager.cpp]: I feel like given the noisiness of IMUs and the frame rate, we should be using a running average.  I suspect that if we did a graph of the velocity while holding the hand still we'd see lots of spikes.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9015e98b708bc1b8b555b3281ecde0bc19c911f2 available at: https://gist.github.com/505160280c103cc8a197<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6157.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6157.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3e4bada313c65e485a35d014ee11dad213b46ef2 available at: https://gist.github.com/505160280c103cc8a197<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6157.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-6157.dmg

**Pull request merged**